code/modules/mob/living/carbon/human/species_types/dullahan.dm

```diff
@@ -0,0 +1,108 @@
+/datum/species/dullahan
+	name = "dullahan"
+	id = "dullahan"
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS,NOBREATH)
+	mutant_bodyparts = list("tail_human", "ears", "wings")
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	use_skintones = 1
+	mutant_brain = /obj/item/organ/brain/dullahan
+	mutanteyes = /obj/item/organ/eyes/dullahan
+	mutanttongue = /obj/item/organ/tongue/dullahan
+	mutantears = /obj/item/organ/ears/dullahan
+	blacklisted = 1
+	limbs_id = "human"
+	skinned_type = /obj/item/stack/sheet/animalhide/human
+
+	var/obj/item/bodypart/head/myhead
+
+/datum/species/dullahan/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
+	..()
+	var/obj/item/bodypart/head/head = H.get_bodypart("head")
+	if(head)
+		myhead = head
+		head.drop_limb()
+		myhead.flags_1 = HEAR_1
+		myhead.throwforce = 25
```

Comment:
Out of all things that would not work, THIS in particular isn't going to. Would you risk your life for 25 damage?

Reply:
Big plays

Reply:
Win big or go home